test/fixtures/output/kafka/10.golden

```diff
@@ -1,3 +1,4 @@
-  Cloud ID |     Cloud Name      | Region ID |       Region Name        
------------+---------------------+-----------+--------------------------
-  aws      | Amazon Web Services | us-east-1 | us-east-1 (N. Virginia)  
+  Cloud ID |     Cloud Name      |   Region ID    |       Region Name        
+-----------+---------------------+----------------+--------------------------
+  aws      | Amazon Web Services | ap-northeast-1 | ap-northeast-1 (Tokyo)   
```

Comment:
I assume this entry is showing up now because it was previously marked as not schedulable. Can you clean up instances of that field from the test server?